packages/relic/test/ssl/ssl_certs.dart

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Question: This is the file that should probably be ignored podspec ignored right?